test/cases/api.sh

```diff
@@ -145,6 +145,23 @@ function instanceWaitSSH() {
   done
 }
 
+function instanceCheck() {
+  local _ssh="$1"
+
+  # Check if postgres is installed
+  $_ssh rpm -q postgresql
+
+  # Verify subscribe status
+  subscribe_org_id=$($_ssh sudo subscription-manager identity | grep 'org ID')
+  [[ "$subscribe_org_id" == "org ID: $API_TEST_SUBSCRIPTION_ORG_ID" ]]
+
+  # Verify yum install a small package
+  $_ssh sudo dnf -y install dos2unix
```

Comment:
I would suggest that first we check with rpm -q that dos2unix is not installed before attempting to install it with dnf b/c dnf returns 0 if the package is already present.

Reply:
Thanks @atodorov ! Updated.